scripts/pre_push_hook_test.py

```diff
@@ -621,13 +621,16 @@ def mock_check_output(cmd_tokens):
         with self.get_remote_name_swap, self.get_refs_swap, self.print_swap:
             with self.collect_files_swap, self.uncommitted_files_swap:
                 with check_output_swap, self.assertRaisesRegexp(
-                    SystemExit, '1'):
+                    SystemExit, '1'
+                ):
                     with self.swap_check_backend_python_libs:
                         pre_push_hook.main(args=[])
-        self.assertTrue(
-            '\nCould not change branch to branch2. This is most probably '
+        self.assertIn(
+            '\nCould not change branch to branch1. This is most probably '
```

Comment:
How come branch2 changed to branch1 in the error message?

Reply:
Both of these are correct, the thing is that ordering of dictionary keys (or items) works differently, so `branch1` is returned earlier.